src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/RazorFormattingPassBase.cs

```diff
@@ -14,15 +15,14 @@
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
-namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting;
+namespace Microsoft.CodeAnalysis.Razor.Formatting;
 
-internal sealed class RazorFormattingPass(
-    IDocumentMappingService documentMappingService,
-    RazorLSPOptionsMonitor optionsMonitor)
+using SyntaxNode = AspNetCore.Razor.Language.Syntax.SyntaxNode;
```

Comment:
Unfortunately there is already an actual RazorSyntaxNode - 

https://github.com/dotnet/razor/blob/main/src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Syntax/RazorSyntaxNode.cs

I could still move up and pick a different alias (suggestions? I think David for a while was using R prefix to mean "Roslyn", we could maybe use Rzr, or just not move this one I guess).

Reply:
No abbreviations please. "Razor" and "Roslyn". "Lsp" is acceptable because it isn't an abbreviation. :smile:

If David was using "R" as prefix to mean "Roslyn", he should be lectured. :smile:

Reply:
Maybe I'm misremembering :) OK, so any suggestions on type name alias for Razor SyntaxNode that's not RazorSyntaxNode? :)

Reply:
I'll leave it as is for now, we can follow up later if we merge Razor SyntaxNode and RazorSyntaxNode